examples/bzlmod/entry_point/test_entry_point.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pathlib
+import subprocess
+import unittest
+
+
+class ExampleTest(unittest.TestCase):
+    def test_entry_point(self):
+        rlocation_path = os.environ.get("YAMLLINT_ENTRY_POINT")
+        assert rlocation_path is not None
+
+        entry_point = pathlib.Path("..") / rlocation_path
```

Comment:
This would be a great place to use `//python/runfiles` instead of just accessing the file directly as the latter probably doesn't work on Windows.